base/error.jl

```diff
@@ -78,7 +78,29 @@ systemerror(p, b::Bool; extrainfo=nothing) = b ? throw(Main.Base.SystemError(str
 
 ## assertion functions and macros ##
 
+
+"""
+    assert(cond)
+
+Throw an [`AssertionError`](@ref) if `cond` is `false`.
+Also available as the macro `@assert expr`.
```

Comment:
I'm kind of busy rn, if you want to edit the PR to change this (it's a good idea) go wild :)